src/GHC.hs

```diff
@@ -4,7 +4,7 @@ module GHC (
     deepseq, deriveConstants, directory, dllSplit, filepath, genapply,
     genprimopcode, ghc, ghcBoot, ghcCabal, ghci, ghcPkg, ghcPrim, ghcTags,
     ghcSplit, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc, hpcBin, integerGmp,
-    integerSimple, iservBin, libffi, mkUserGuidePart, parallel, pretty,
+    integerSimple, iservBin, libffi, mkUserGuidePart, GHC.parallel, pretty,
```

Comment:
IMHO it would be better to just hide `parallel` from the `Shake` import and accept the warning for a while than uglify the export lists for eternity.